### ISSUE TITLE
[BUG] Pulsar returns an error when searching for `web`

### ISSUE THREAD

**Issue by Daeraxa** [pulsar-edit/package-backend#88]
**Is this Bug Present in the upstream API Server?**
- [ ] Yes
- [ ] No
- [x] Unsure

**What is the Bug**
When searching for `web` in the Pulsar "Install Packages" panel it errors:
```
Searching for “web” failed.Hide output…
Cannot convert undefined or null to object
[{"name":"web-view","readme":"# Atom Web View...
```
The returned json from `https://api.pulsar-edit.dev/api/packages/search?q=web` appears to be valid so I'm wondering if Pulsar is expecting to have a field that is null in the returned JSON?

**How to Replicate the Bug**
- Open "Install Packages"
- Type "web"

**Comment by savetheclocktower** [comment]
The original reporter narrowed it down to the `web-developer-tools` package. Trying to [view this package](https://web.pulsar-edit.dev/packages/web-developer-tools) (or search for it) on the frontend also returns an error.

**Comment by Daeraxa** [comment]
It actually has no metadata at all which makes me think it is indeed malformed.

```json
    {
        "main": "./lib/web-tools",
        "name": "web-developer-tools",
        "engines": {
            "atom": ">=1.0.0 <2.0.0"
        },
        "license": "MIT",
        "version": "0.4.4",
        "keywords": [],
        "repository": "https://github.com/manngo/atom-web-tools",
        "description": "A Set of Tools to work with Web Development",
        "dependencies": {},
        "activationCommands": {
            "atom-workspace": [
                "web-tools:about",
                "web-tools:html:lists",
                "web-tools:html:structure",
                "web-tools:tables",
                "web-tools:html:paragraphs",
                "web-tools:html:paragraphs-headings",
                "web-tools:html:insert",
                "web-tools:html:anchors",
                "web-tools:html:html5",
                "web-tools:css:colors",
                "web-tools:misc:spaces-tabs",
                "web-tools:misc:smart-quotes",
                "web-tools:misc:entify",
                "web-tools:misc:js-chars",
                "web-tools:misc:collapse-lines",
                "web-tools:misc:getPath",
                "web-tools:misc:copyProfile",
                "web-tools:misc:sample",
                "web-tools:misc:replace"
            ]
        },
        "downloads": "795",
        "stargazers_count": "0",
        "releases": {
            "latest": "0.4.4"
        }
    },
```

**Comment by confused-Techie** [comment]
As we now have an issue opened for the specific package that has malformed data, and since the frontend totally failing to return anything has been resolved in [`package-frontend#75](https://github.com/pulsar-edit/package-frontend/pull/75). I'll go ahead and close this issue.

Thanks for everyone helping out on this one and for reporting it
